.gitlab-ci.yml

```diff
@@ -1,6 +1,6 @@
 workflow:
   rules:
-    - if: $CI_COMMIT_MESSAGE =~ /build: publish/
+    - if: $CI_COMMIT_MESSAGE =~ /publish/
```

Comment:
"publish" is not an uncommon word in programming, esp in a codebase with pubsub in it.  These doesn't seem specific enough.

Reply:
good point. i'll fix once i can get the regex passed the ci linter